README.md

```diff
@@ -170,6 +170,8 @@ To specify Tenjin as the destination for your [SK Ad Network postbacks](https://
 
 These steps are adapted from Apple's instructions at [https://developer.apple.com/documentation/storekit/skadnetwork/configuring_an_advertised_app](https://developer.apple.com/documentation/storekit/skadnetwork/configuring_an_advertised_app).
 
+**NOTE**: If you are using AppLovin MAX for mediation, their SDK will overwrite Tenjin URL with their URL during the app build process. In that case, AppLovin can forward us the postbacks, so please contact your account manager to enable the forwarding.
```

Comment:
This is specific to Unity, right? Should we have the instructions in the Unity SDK instead?

Reply:
Hmm let me check if this is also the case for Native iOS.

Reply:
We don't need this for Native iOS. Going to close this ticket.